### PR TITLE
Modify the function to return data from CRUD operations.

### PR DESCRIPTION
**Pull request by Filldexnoi**
Modify the function to return data from CRUD operations to verify correctness and facilitate testing.

### CODE REVIEW

File: Handler/handler_stock.go [hunk 1]
```diff
@@ -60,21 +61,23 @@ func (h *StockHandler) UpdateStock(c *fiber.Ctx) error {
...
 		return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{"error": err.Error()})
 	}
-	err = h.UseCase.UpdateStock(stock.ToStock(), uint(id))
+	stockEntity, err := h.UseCase.UpdateStock(stock.ToStock(), uint(id))
 	if err != nil {
 		return c.Status(fiber.StatusInternalServerError).JSON(fiber.Map{"error": err.Error()})
 	}
-	return c.JSON(stock)
+	ResStock := payload.StockToStockRes(stockEntity)
+	return c.JSON(ResStock)
```
Comment (kimzey) [inline Handler/handler_stock.go]: last return in func handler it no status ?

 c.JSON(<data>)  >>>  c.Status(<status>).JSON(<data>)

File: entities/Order.go [hunk 2]
```diff
@@ -58,3 +58,17 @@ func (o *Order) InitStatus() (*Order, error) {
 
 	return o, fmt.Errorf("%w: from %s to %s", errors.New("invalid order status"), o.Status, "New")
 }
+
+func (o *Order) IsEqualCreatedOrder(createdOrder *Order) bool {
```
Comment (kimzey) [inline entities/Order.go]: Why you have or use Func IsEqual+service+entities ?

**Review by kimzey — COMMENTED** [review]
last return in func handler no status ?

 c.JSON(<data>) > c.Status(<status>).JSON(<data>)

File: entities/Order.go (hunk 2 above)
Comment (Filldexnoi) [reply entities/Order.go]: now this function was deleted

Comment (Filldexnoi) [reply entities/Order.go]: Fixed it. Please check again.

File: Handler/handler_stock.go (hunk 1 above)
Comment (Filldexnoi) [reply Handler/handler_stock.go]: Fixed it. Please check again.

**Review by kimzey — APPROVED** [review]
OK

**Pull request merged**